hardware/MQTT.cpp

```diff
@@ -3593,14 +3666,8 @@ bool MQTT::SendSwitchCommand(const std::string &DeviceID, const std::string &Dev
 				if (!result.empty())
 				{
 					_eSwitchType sSwitchType = (_eSwitchType)atoi(result[0][0].c_str());
-					if (
```

Comment:
Could you undo the formatting here. It makes the code harder to read

Reply:
Same again...